### PR TITLE
ko: Keep up with upstream

### PR DESCRIPTION
**Pull request by gochist**
Partly fixes_ #15001

1 file deferred from previous team milestone

**These should be compared with `dev-1.14-ko.4`**
- [x] content/en/docs/setup/_index.md
- [x] content/en/docs/setup/minikube.md -> content/en/docs/setup/learning-environment/minikube.md
- [x] content/en/docs/tutorials/online-training/overview.md

9 files to be renamed

- [x] R088 content/en/docs/setup/certificates.md -> content/en/docs/setup/best-practices/certificates.md
- [x] R099 content/en/docs/setup/cluster-large.md -> content/en/docs/setup/best-practices/cluster-large.md
- [x] R099 content/en/docs/setup/multiple-zones.md -> content/en/docs/setup/best-practices/multiple-zones.md
- [x] R098 content/en/docs/setup/node-conformance.md -> content/en/docs/setup/best-practices/node-conformance.md
- [x] R067 content/en/docs/setup/minikube.md -> content/en/docs/setup/learning-environment/minikube.md
- [x] R097 content/en/docs/setup/cri.md -> content/en/docs/setup/production-environment/container-runtimes.md
- [x] R073 content/en/docs/setup/on-premises-vm/_index.md -> content/en/docs/setup/production-environment/on-premises-vm/_index.md
- [x] R099 content/en/docs/setup/custom-cloud/kops.md -> content/en/docs/setup/production-environment/tools/kops.md
- [x] R078 content/en/docs/setup/turnkey/_index.md -> content/en/docs/setup/production-environment/turnkey/_index.md

3 files to be deleted 
- [x] content/en/docs/setup/custom-cloud/_index.md
- [x] content/en/docs/setup/independent/_index.md
- [x] content/en/docs/setup/release/building-from-source.md

/hold

### CODE REVIEW

File: content/ko/docs/setup/_index.md [hunk 1]
```diff
@@ -1,76 +1,106 @@
...
+{{% capture body %}}
 
-{{% /capture %}}
+## 학습 환경
+
+쿠버네티스를 배우고 있다면, 쿠버네티스 커뮤니티에서 지원하는 도구나, 로컬 머신에서 쿠버네티스를 설치하기 위한 생태계 내의 도구와 같은 도커 기반의 솔루션을 사용하자.
+
+{{< table caption="쿠버네티스를 배포하기 위해 커뮤니티와 생태계에서 지원하는 도구를 나열한 로컬 머신 솔루션 표." >}}
+
+|Community           |Ecosystem     |
```
Comment (sftim) [inline content/ko/docs/setup/_index.md]: These headings are untranslated; is that intended?

File: content/ko/docs/setup/_index.md [hunk 2]
```diff
@@ -1,76 +1,106 @@
...
+몇 가지 가능한 쿠버네티스 클러스터의 추상적인 개념은 {{< glossary_tooltip text="애플리케이션" term_id="applications" >}}, {{< glossary_tooltip text="데이터 플레인" term_id="data-plane" >}}, {{< glossary_tooltip text="컨트롤 플레인" term_id="control-plane" >}}, {{< glossary_tooltip text="클러스터 인프라스트럭처" term_id="cluster-infrastructure" >}}, 및 {{< glossary_tooltip text="클러스터 운영" term_id="cluster-operations" >}}이다.
+
+다음의 다이어그램은 쿠버네티스 클러스터에 대해 가능한 추상화와 스스로 관리해야하는지 제공자에 의해 관리되는지를 나열한다.
+
+운영 환경 솔루션![운영 환경 솔루션](/images/docs/KubernetesSolutions.svg)
+
+{{< table caption="제공자와 솔루션을 나열한 운영 환경 솔루션 표." >}}
+다음 운영 환경 솔루션 표는 제공자와 솔루션을 나열한다.
+
+|Providers  | Managed | Turnkey cloud  | On-prem datacenter  | Custom (cloud) | Custom (On-premises VMs)| Custom (Bare Metal) |
```
Comment (sftim) [inline content/ko/docs/setup/_index.md]: These headings are untranslated; is that intended?

File: content/ko/docs/setup/_index.md [hunk 3]
```diff
@@ -1,76 +1,106 @@
...
+| [Cisco Container Platform](https://cisco.com/go/containers)     |       |  | &#x2714;  |  | |
+| [Cloud Foundry Container Runtime (CFCR)](https://docs-cfcr.cfapps.io/)       |     |  |   | &#x2714; |&#x2714; |
+| [CloudStack](https://cloudstack.apache.org/)           |      |  |   | | &#x2714;|
+| [Canonical](https://www.ubuntu.com/kubernetes/docs/quickstart)      |              | &#x2714;       |             | &#x2714;     |&#x2714;  | &#x2714;
+| [Containership](https://containership.io/containership-platform)            | &#x2714;       |&#x2714;  |     |     |                   |
+| [Digital Rebar](https://provision.readthedocs.io/en/tip/README.html)            |        |  |      |     |  | &#x2714;
+| [DigitalOcean](https://www.digitalocean.com/products/kubernetes/)        | &#x2714;      |  |      |     |            |
+| [Docker Enterprise](https://www.docker.com/products/docker-enterprise) |       |&#x2714;  | &#x2714;     |  |     | &#x2714;
+| [Fedora (Multi Node)](https://kubernetes.io/docs/getting-started-guides/fedora/flannel_multi_node_cluster/)     |      |  |      | |    &#x2714;       | &#x2714;
```
Comment (sftim) [inline content/ko/docs/setup/_index.md]: How about translating “Multi Node” here?

File: content/ko/docs/setup/_index.md (hunk 1 above)
Comment (gochist) [reply content/ko/docs/setup/_index.md]: @sftim Yes, just for my convenience. Now I has translated them. Thank you for your kind review!

**Comment by gochist** [comment]
@sftim Thank you! I addressed all your comments.

File: content/ko/docs/setup/_index.md (hunk 1 above)
Comment (sftim) [reply content/ko/docs/setup/_index.md]: No worries @gochist

File: content/ko/docs/setup/_index.md [hunk 4]
```diff
@@ -1,76 +1,106 @@
...
+|                     | [MicroK8s](https://microk8s.io/)|
+|                     | [IBM Cloud Private-CE (Community Edition)](https://github.com/IBM/deploy-ibm-cloud-private) |
+|                     | [IBM Cloud Private-CE (Community Edition) on Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers)|
+|                     | [k3s](https://k3s.io)|
+|                     | [Ubuntu on LXD](/docs/getting-started-guides/ubuntu/)|
+
+
+## 운영 환경
+
+운영 환경을 위한 솔루션을 평가할 때에는, 쿠버네티스 클러스터(또는 _추상적인 개념_) 운영을 스스로 관리하기를 원하는지, 제공자에게 넘기기를 원하는지 고려하자.
```
Comment (seokho-son) [inline content/ko/docs/setup/_index.md]: 원문: When evaluating a solution for a production environment, consider which aspects of operating a Kubernetes cluster (or _abstractions_) you want to manage yourself or offload to a provider.

원문을 고려하여 다음과 같이 제안 드립니다.
```suggestion
운영 환경을 위한 솔루션을 평가할 때에는, 쿠버네티스 클러스터 운영에 대한 어떤 측면(또는 _추상적인 개념_)을 스스로 관리하기를 원하는지, 제공자에게 넘기기를 원하는지 고려하자.
```

File: content/ko/docs/setup/_index.md [hunk 5]
```diff
@@ -1,76 +1,106 @@
...
+
+몇 가지 가능한 쿠버네티스 클러스터의 추상적인 개념은 {{< glossary_tooltip text="애플리케이션" term_id="applications" >}}, {{< glossary_tooltip text="데이터 플레인" term_id="data-plane" >}}, {{< glossary_tooltip text="컨트롤 플레인" term_id="control-plane" >}}, {{< glossary_tooltip text="클러스터 인프라스트럭처" term_id="cluster-infrastructure" >}}, 및 {{< glossary_tooltip text="클러스터 운영" term_id="cluster-operations" >}}이다.
+
+다음의 다이어그램은 쿠버네티스 클러스터에 대해 가능한 추상화와 스스로 관리해야하는지 제공자에 의해 관리되는지를 나열한다.
```
Comment (seokho-son) [inline content/ko/docs/setup/_index.md]: 조금 더 자연스럽게 변경해보았습니다. 참고하시기 바랍니다.

원문: The following diagram lists the possible abstractions of a Kubernetes cluster and whether an abstraction is self-managed or managed by a provider.

```suggestion
다음의 다이어그램은 쿠버네티스 클러스터에 대해 가능한 추상적인 개념을 나열하고, 각 추상적인 개념을 사용자 스스로 관리하는지 제공자에 의해 관리되는지를 보여준다.
```

File: content/ko/docs/setup/_index.md [hunk 6]
```diff
@@ -1,76 +1,106 @@
...
+
+운영 환경 솔루션![운영 환경 솔루션](/images/docs/KubernetesSolutions.svg)
+
+{{< table caption="제공자와 솔루션을 나열한 운영 환경 솔루션 표." >}}
+다음 운영 환경 솔루션 표는 제공자와 솔루션을 나열한다.
+
+|제공자  | 매니지드 | 턴키 클라우드  | 온-프렘(on-prem) 데이터센터  | 커스텀 (클라우드) | 커스텀 (On-premises VMs)| 커스텀 (베어 메탈) |
```
Comment (seokho-son) [inline content/ko/docs/setup/_index.md]:
```suggestion
|제공자  | 매니지드 | 턴키 클라우드  | 온-프렘(on-prem) 데이터센터  | 커스텀 (클라우드) | 커스텀 (온-프레미스 VMs)| 커스텀 (베어 메탈) |
```

File: content/ko/docs/tutorials/online-training/overview.md [hunk 7]
```diff
@@ -17,33 +17,31 @@ content_template: templates/concept
 
 * [Cloud Native Certified Kubernetes Administrator (CKA) with Hands-On Labs & Practice Exams (Linux Academy)](https://linuxacademy.com/linux/training/course/name/cloud-native-certified-kubernetes-administrator-cka)
 
-* [Certified Kubernetes Administrator Developer 준비 과정 및 모의 시험 (KodeKloud)](https://kodekloud.com/p/certified-kubernetes-administrator-with-practice-tests)
+* [Certified Kubernetes Administrator Preparation Course with Practice Tests (KodeKloud)](https://kodekloud.com/p/certified-kubernetes-administrator-with-practice-tests)
```
Comment (seokho-son) [inline content/ko/docs/tutorials/online-training/overview.md]: 기존의 번역된 내용도 다시 영문으로 바뀐 경우가 있는데, 고민해서 처리하신 것이라 짐작하였습니다 ~~

**Comment by gochist** [comment]
@seokho-son 감사합니다! 리뷰 반영했고, #15099 에 의해 발생한 Conflict도 추가로 해결했습니다.

**Comment by seokho-son** [comment]
@gochist 감사합니다 ~~
/lgtm

@ClaudiaJKang 검토해보시고 승인 부탁드립니다 ~~ :)

**Pull request merged**